scripts/invocation_upload.py

```diff
@@ -0,0 +1,179 @@
+import asyncio
+import logging
+import os
+import tempfile
+from typing import List
+
+import pandas as pd
+import requests
+from motor.motor_asyncio import AsyncIOMotorClient
+from odmantic import AIOEngine
+from pydantic import ValidationError
+
+from osm._utils import get_compute_context_id
+from osm.schemas import Client, Invocation, Work
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+)
+logger = logging.getLogger(__name__)
+
+# NOTICE: items without mapping to the dataframe
+unmapped = {
+    "article": "",
+    "is_relevant": None,
+    "is_explicit": None,
+    "user_comment": "",
+    "osm_version": "",
+}
```

Comment:
@leej3 I retained the unmapped dictionary because some of the values are needed outside the metrics. the unmapped dic holds all missing values and are referenced in the appropriate location